contentctl/actions/detection_testing/views/DetectionTestingView.py

```diff
@@ -94,47 +101,68 @@ def getSummaryObject(
                 test_job_fields=test_job_fields, test_result_fields=test_result_fields
             )
 
+            # TODO (cmcginley): I find the enum to string conversion we use to be v confusing
             # Aggregate detection pass/fail metrics
-            if summary["success"] is False:
+            if detection.test_status == TestResultStatus.FAIL:
                 total_fail += 1
+            elif detection.test_status == TestResultStatus.PASS:
+                total_pass += 1
+            elif detection.test_status == TestResultStatus.SKIP:
+                total_skipped += 1
+
+            # Aggregate production status metrics
+            if detection.status == DetectionStatus.production.value:                                # type: ignore
```

Comment:
Could we think about disabling the auto conversion of enums to their values? It leads to typing weirdness and some inconsistent logical checks. Unsure if there's a way to serialize the value as a string, but have it function internally as a proper enum

Reply:
I like this idea a lot. I'd say its non-critical quality of life type change, so I'm fine with just capturing it as an issue for now. But I think it will make the code a lot easier to understand, esp for someone new to the codebase

If you agree, I can make an issue and then decorate the instances of `use_enum_values` with the issue number in a comment

Reply:
Yes, I agree with both of these. When you have time, please go ahead and create the issue + add TODO.

Reply:
https://github.com/splunk/contentctl/issues/266

Added appropriate TODOs in most recent commit